packages/eas-cli/src/commandUtils/context/__tests__/ProjectDirContextField-test.ts

```diff
@@ -23,31 +25,13 @@ describe(ProjectDirContextField['findProjectRootAsync'], () => {
       },
       '/app'
     );
-    await expect(ProjectDirContextField['findProjectRootAsync']({ cwd: '/app' })).rejects.toThrow(
+    await expect(ProjectDirContextField['findProjectRootAsync']()).rejects.toThrow(
```

Comment:
If I understand correctly process.cwd is preserved inside the test, so detecting project root here fails not because there is no packagae.json in the directory, but because cwd is totally outside the `/app` dir.